packages/core/src/shadow/CascadedShadowCasterPass.ts

```diff
@@ -238,6 +239,8 @@ export class CascadedShadowCasterPass extends PipelinePass {
 
         opaqueQueue.render(context, PipelineStage.ShadowCaster);
         alphaTestQueue.render(context, PipelineStage.ShadowCaster);
+        // Reset stencil
+        scene._maskManager.drawMask(context, PipelineStage.ShadowCaster, SpriteMaskLayer.Nothing);
```

Comment:
Currently there is no need to do this part because `SpriteMaskShader` does not contain `ShadowCasterPass`

Reply:
done